lib/client.js

```
@@ -129,6 +130,33 @@ class Client extends Entity {
   get serviceName() {
     return this._serviceName;
   }
+
+  /**
+   * Configure introspection.
+   * @param {Clock} clock - Clock to use for service event timestamps
+   * @param {QoS} qos - QoSProfile for the service event publisher
+   * @param {ServiceIntrospectionState} introspectionState - State to set introspection to
+   */
+  configureIntrospection(clock, qos, introspectionState) {
+    if (DistroUtils.getDistroId() <= DistroUtils.getDistroId('humble')) {
+      console.warn(
+        'Service introspection is not supported by this versionof ROS 2'
+      );
+      return;
```

How about throwing an error instead of just returning here?